tests/virtiofs/datavolume.go

```diff
@@ -251,34 +287,55 @@ var _ = Describe("[sig-storage] virtiofs", decorators.SigStorage, func() {
 				}
 			}
 			Expect(foundContainer).To(BeTrue())
-		})
+		},
+			Entry("unprivileged virtiofsd", util.NamespaceTestDefault),
+			Entry("privileged virtiofsd", testsuite.NamespacePrivileged),
+		)
 	})
 
 	Context("Run a VMI with VirtIO-FS and a datavolume", func() {
-		var dataVolume *cdiv1.DataVolume
+		var (
+			dataVolume        *cdiv1.DataVolume
+			isVirtioFSEnabled bool
+			sc                string
+			exists            bool
+		)
+
 		BeforeEach(func() {
-			checks.SkipTestIfNoFeatureGate(virtconfig.VirtIOFSGate)
+			isVirtioFSEnabled = checks.HasFeature(virtconfig.VirtIOFSGate)
 			if !libstorage.HasCDI() {
 				Skip("Skip DataVolume tests when CDI is not present")
 			}
 
-			sc, exists := libstorage.GetRWOFileSystemStorageClass()
+			sc, exists = libstorage.GetRWOFileSystemStorageClass()
 			if !exists {
 				Skip("Skip test when Filesystem storage is not present")
 			}
-
-			dataVolume = libdv.NewDataVolume(
-				libdv.WithRegistryURLSource(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine)),
-				libdv.WithPVC(libdv.PVCWithStorageClass(sc)),
-			)
 		})
 
 		AfterEach(func() {
 			libstorage.DeleteDataVolume(&dataVolume)
+			if isVirtioFSEnabled {
```

Comment:
This is redundant, the testsuite reverts the KV. This is rendering the `isVirtioFSEnabled` redundant.

Reply:
done